draft-dns-zone-digest.xml

```diff
@@ -429,12 +428,14 @@ example.com. 86400 IN &RRNAME; ( 2018031500 4 FEBE3D4CE2EC2FFA4BA9
         <t>
           Following addition of the placeholder record, the zone MAY be signed with DNSSEC.
           Note that when the digest calculation is complete, and the &RRNAME; record is updated,
-          the signature(s) for that record MUST be recalculated and updated as well.
+          the signature(s) for that record MUST be recalculated and
+          updated as well.  (Alternatively, the calculation of that
+          RRSIG value can be delayed until after the steps below are completed).
```

Comment:
I feel like the new parenthetical remark is redundant?  The start of the sentence says "when the digest calculation is complete" and the new text says "steps below are completed".  How are these different?